src/js/configs.js

```diff
@@ -43,6 +43,15 @@ module.exports = {
     tls13: '0.11.5',
     usesOpenssl: false,
   },
+  coturn: {
+    highlighter: 'ini',
+    latestVersion: '4.6.2',
+    name: 'Coturn',
+    showSupports: false,
+    supportsHsts: false,
+    supportsOcspStapling: false,
+    tls13: '3.2.2',
```

Comment:
It looks like TLS 1.3 support was added via https://github.com/coturn/coturn/pull/989 

I don't see it mentioned in https://github.com/coturn/coturn/blob/master/ChangeLog before this point. Can you point me to where you found that TLS 1.3 was supported in coturn 3.2.2?

Reply:
I don't find anymore where I found this. So it's safest to use the version that publish https://github.com/coturn/coturn/pull/989 : 4.6.2